articles/sql.md

```diff
@@ -0,0 +1,86 @@
+---
+authorName: Ognjen Blagojević
+authorGithubUsername: ognjen-blagojevic-zuehlke
+issue: 85
+title: Do something about that slow SQL query
+---
+# {{page.title}}
+
+Solving the problem of poor database performance is the topic on which the
+books are written, but we will here briefly describe the procedure how to 
```

Comment:
Please use a single line per paragraph (applies to the whole article)

Reply:
Done